crates/node/src/lib.rs

```diff
@@ -1268,120 +1174,94 @@ impl Node {
             return Err(MutateCallError::NoConnectedPeers);
         }
 
-        self.push_action(context.id, PeerAction::Transaction(transaction.clone()))
+        self.push_action(context.id, PeerAction::ActionList(action_list))
             .await
             .map_err(|err| {
                 error!(%err, "Failed to push transaction over the network.");
                 MutateCallError::InternalError
             })?;
 
-        let tx_hash = self
-            .tx_pool
-            .insert(self.id, transaction, Some(outcome_sender))
-            .map_err(|err| {
-                error!(%err, "Failed to insert transaction into the pool.");
-                MutateCallError::InternalError
-            })?;
-
-        Ok(tx_hash)
+        Ok(())
     }
 
+    // TODO: Should we be using this somewhere in the CRDT approach, or not?
     async fn execute_in_context(
```

Comment:
this was supposed to be called after receiving a coordinator confirmation, it's no longer needed, can be removed

Reply:
Sounds good, I agree with that 👍 I wanted to be careful.